Xamarin.Forms.Platform.UAP/KeyboardExtensions.cs

```diff
@@ -42,8 +44,39 @@ public static InputScope ToInputScope(this Keyboard self)
 			}
 			else
 			{
-				name.NameValue = InputScopeNameValue.Default;
+				var custom = (CustomKeyboard)self;
+				bool capitalizedSentenceEnabled = custom.Flags.HasFlag(KeyboardFlags.CapitalizeSentence);
+				bool capitalizedWordsEnabled = custom.Flags.HasFlag(KeyboardFlags.CapitalizeWord);
+				bool capitalizedCharacterEnabled = custom.Flags.HasFlag(KeyboardFlags.CapitalizeCharacter);
+				bool spellcheckEnabled = custom.Flags.HasFlag(KeyboardFlags.Spellcheck);
+				bool suggestionsEnabled = custom.Flags.HasFlag(KeyboardFlags.Suggestions);
+				InputScopeNameValue nameValue = InputScopeNameValue.Default;
+				
+				if(capitalizedSentenceEnabled)
```

Comment:
👮 Noticing inconsistency with the space between `if` and `(`. Just wanted to point it out so you can fix your IDE. :)